### PR TITLE
[SET-46] Update Docs

### PR DESCRIPTION
**Pull request by dallascrichmond**
## 🎯 Summary
- Create Swagger documentation for the current endpoints
- Will need to update further without appropriate responses
- Moved docs to docs folder instead of public folder

[SET-46](https://citz-imb.atlassian.net/jira/software/c/projects/SET/boards/51?selectedIssue=SET-46)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: api/routes/inquiry-router.ts
Comment (SDToews) [inline api/routes/inquiry-router.ts:1]: please change the endpoints for the inquiry/guid to be consistent for get and patch

**Review by SDToews — CHANGES_REQUESTED** [review]
please see comment for inquiry-router.ts file

File: api/routes/inquiry-router.ts
Comment (dallascrichmond) [reply api/routes/inquiry-router.ts:1]: The inquiry/guid route has no PATCH as it only returns all inquiries that are associated with the user via their guid. The inquiry/:id route has a PATCH and will have a GET that will update or return a single inquiry via it's id.

Comment (SDToews) [reply api/routes/inquiry-router.ts:1]: got it, thanks!

**Review by SDToews — APPROVED** [review]

**Pull request merged**